implementations/rust/ockam/ockam_api/src/cli_state.rs

```diff
@@ -735,6 +735,15 @@ impl NodeState {
         Ok(())
     }
 
+    pub fn is_running(&self) -> bool {
+        if let Ok(Some(pid)) = self.pid() {
+            let system_path = PathBuf::from(format!("/proc/{}/status", pid));
```

Comment:
This would only work for linux... you can try with `sysinfo`. It has a method to iterate through the running processes that you can filter.

Reply:
Yes, that make sense. Thank you for the guidance. I just pushed the changes with `sysinfo`. Let me know what do you think.